modules/core/src/main/java/org/apache/ignite/internal/util/future/GridFinishedFuture.java

```diff
@@ -160,6 +160,11 @@ public GridFinishedFuture(Throwable err) {
         return fut;
     }
 
+    /** {@inheritDoc} */
+    @Override public IgniteInternalFuture chainCompose(IgniteClosure doneCb) {
```

Comment:
It seems to me, it makes sense to implement this method for `GridFinishedFuture`. What do you think?

Reply:
seems to be easy, i will